packages/desktop-client/src/components/sidebar/ItemContent.tsx

```diff
@@ -6,8 +6,8 @@ import View from '../common/View';
 
 type ItemContentProps = {
   style: CSSProperties;
-  to: string;
-  onClick: MouseEventHandler<HTMLDivElement>;
+  to?: string;
+  onClick?: MouseEventHandler<HTMLDivElement>;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏is it that there can be either `to` or `onClick`? But not both at the same time? If so - we could solve it by changing the types slightly.

(very poor naming chosen here intentionally as this is just an example)

```
type BaseProps = { style: CSSProperties };
type ItemContentProps = AProps | BProps;

type AProps = BaseProps & { to: string };
type BProps = BaseProps & { onClick: () => void }; 
```